authui/src/authflowv2/components.css

```diff
@@ -6,6 +6,7 @@
 @import "./components/secondary-button.css";
 @import "./components/tertiary-button.css";
 @import "./components/label-button.css";
+@import "./components/inline-button.css";
```

Comment:
I think link-button is a better name. @tung2744 Thoughts?

Reply:
Maybe simply call it resend button, because it is actually only for resending and not a generic component in figma. 🤔

Reply:
Arvin said it is used in other places too.

Reply:
It is not a component in figma.
I saw the same style here (The login button): https://www.figma.com/file/LRE8EfCaTfYxC0lk5AeMmE/New-Auth-UI?type=design&node-id=762-37040&mode=design&t=HdwuDJWdD4Lf2u1u-4 
But they are having different purpose, this one is really inline link / button inside a body text.
I've implemented it in https://github.com/authgear/authgear-server/pull/3611

Reply:
In translation.json, we have some translation containing `<a class="link">`. By using `link`, the element should look like a styled link in the design. Ideally, if we write `<button class="link">`, then the button should look like a link.

Reply:
I've implemented link in body text by styling `<a>` inside body-text in the signup screen pr.